src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -20,6 +20,7 @@
 
 import java.util.List;
 
+import ch.njol.skript.util.*;
```

Comment:
Please disable wild card imports in your IDE. We can't be importing classes we don't need.